rest-mvc/impl/src/main/java/com/intuit/tank/rest/mvc/rest/cloud/JobEventSender.java

```diff
@@ -248,6 +248,10 @@ private List<String> getInstancesForJob(String jobId) {
         CloudVmStatusContainer statuses = vmTracker.getVmStatusForJob(jobId);
         if (statuses != null) {
             instanceIds = statuses.getStatuses().stream().map(CloudVmStatus::getInstanceId).collect(Collectors.toList());
+            String commaSeparatedIds = statuses.getStatuses().stream()
+                    .map(CloudVmStatus::getInstanceId)
+                    .collect(Collectors.joining(","));
+            LOG.info("JobEventSender getInstancesForJob yields {} for jobId {}", commaSeparatedIds, jobId);
```

Comment:
This feels repudiative. Can't you just make the instanceIds from the step above into a csv string?